tests/test_parser_yaml.py

```diff
@@ -0,0 +1,310 @@
+import unittest
+import sys
+import os
+from datetime import datetime
+
+sys.path.insert(0,
+                os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from config_lib.parsers.parser_yaml import (parse_yaml_string, YAMLSyntaxError,
+                                            _parse_yaml_scalar,
+                                            _parse_yaml_lines)
```

Comment:
There is no need to use modules for working with system parameters and paths here. Since the tests directory is initialized as a separate module, you can directly import the required parser from another directory of the project. It’s better to do it like this: 
```suggestion
import unittest
from datetime import datetime
from config_lib.parsers.parser_yaml import (parse_yaml_string, YAMLSyntaxError,
                                            _parse_yaml_scalar,
                                            _parse_yaml_lines)
```